packages/h3/src/H3Adapter.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  AppControllerRoute,
+  AppViewRoute,
+  BullBoardQueues,
+  ControllerHandlerReturnType,
+  IServerAdapter,
+  UIConfig,
+} from "@bull-board/api/dist/typings/app";
+import fs from "fs";
+import { createRouter, eventHandler, getRouterParams, setResponseHeader, getQuery } from "h3";
+import ejs from "ejs";
+
+export class H3Adapter implements IServerAdapter {
+  private uiHandler = createRouter();
+  private basePath = "/ui";
+  private bullBoardQueues: BullBoardQueues | undefined;
+  private viewPath: string | undefined;
+  private uiConfig: UIConfig = {};
+
+  public setBasePath(path: string): H3Adapter {
+    this.basePath = path;
+    return this;
+  }
+
+  private getStaticFile(path: string, filename: string) {
+    return fs.readFileSync(`${this.viewPath}${path}/${filename}`, "utf-8");
+  }
+
+  private getContentType(filename: string) {
```

Comment:
We could move this out into a `/helpers/getContentType` as other adapters do something similar.